src/Employer/Employer.Web/ViewModels/Part1/Title/TitleViewModel.cs

```diff
@@ -1,21 +1,22 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
 using Esfa.Recruit.Employer.Web.Configuration.Routing;
 using Esfa.Recruit.Shared.Web.ViewModels;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections;
 
 namespace Esfa.Recruit.Employer.Web.ViewModels.Part1.Title
 {
     public class TitleViewModel : TitleEditModel
     {
         public ReviewSummaryViewModel Review { get; set; } = new ReviewSummaryViewModel();
-
         public IList<string> OrderedFieldNames => new List<string>
         {
-            nameof(Title),
-            nameof(NumberOfPositions)
+            nameof(Title)
         };
-
         public string FormPostRouteName => VacancyId.HasValue ? RouteNames.Title_Post : RouteNames.CreateVacancy_Post;
-
         public PartOnePageInfoViewModel PageInfo { get; set; }
+        public List<VacancySummary> Vacancies { get; set; } = new List<VacancySummary>();
```

Comment:
We don't need all the vacancies.  Change this to a bool `bool HasCloneableVacancies` and BackLink looks at this